src/test/regress/expected/multi_extension.out

```diff
@@ -126,6 +126,9 @@ ALTER EXTENSION citus UPDATE TO '7.0-13';
 ALTER EXTENSION citus UPDATE TO '7.0-14';
 ALTER EXTENSION citus UPDATE TO '7.0-15';
 ALTER EXTENSION citus UPDATE TO '7.1-1';
+ALTER EXTENSION citus UPDATE TO '7.1-2';
```

Comment:
Oh hey, apparently people were forgetting these…

I guess we need to figure out how to refactor the upgrade stuff as it's getting pretty crowded.

Reply:
Agreed, the Makefile is also getting bizarre. Any ideas?

Reply:
For the Makefile, I suppose it wouldn't be tough to enumerate all the versions with a shell expansion/directory listing and create dynamic targets based on them; wasn't worth it initially but I think we passed the "worth it" stage long ago.